### ISSUE TITLE
[Feature Request] Show transfer transactions in one line

### ISSUE THREAD

**Issue by ngocanhtve** [jameskokoska/Cashew#93]
I realized that Cashew is using 2 lines to show a single transfer transaction.

I think just 1 line to show that this account has a cash flow transferred to another account and the amount with a color different from the color of normal transactions is perfect.

For example:

![Screenshot_20231222-144944_Trebuchet](https://github.com/jameskokoska/Cashew/assets/125894401/c0f1e869-69f7-4de6-811f-3b4a3143915e)

**Comment by jameskokoska** [comment]
Since a transfer consists of 2 transactions, it's 2 transaction entries in the database and displayed as such - a transfer in and a transfer out.

**Comment by ngocanhtve** [comment]
> it's 2 transaction entries in the database 

Agreed are 2 different transaction entries in the database. But just showing 1 line is enough to understand. An accountant complained with me about Cashew that 2 lines were unnecessary, his daily accounting needs only 1 line for such a transfer transaction.

And I see other apps also choose to show 1 line for transfer transactions, of course in their database there are still 2 transfer transactions out and in.

**Comment by jameskokoska** [comment]
This has been added in the beta. You can now group transfers by changing the settings of the Balance Correction category. 

See #725 for Beta Testing instructions and information

![image](https://github.com/user-attachments/assets/51cc4b07-839b-4904-9020-da8e3c5eff05)

**Comment by ngocanhtve** [comment]
Thank you so much!
